server.js

```diff
@@ -26,6 +16,12 @@ app.use(require("webpack-dev-middleware")(compiler, {
 }));
 app.use(require("webpack-hot-middleware")(compiler));
 
+//  Create hot-reloading bundle for server rendering
+var basePath = require('path').resolve(__dirname, '.');
+var webpackIsomorphicTools = new WebpackIsomorphicTools(require('./webpack-isomorphic-tools'))
+    .development(true)
+    .server(basePath, function() { });
```

Comment:
Doesn't mean I wouldn't take it once we figure out what's going on! Maybe the creator of that module will help? Failing that, we'll have to poke around the source a bit

> On 17 Oct 2015, at 16:34, Adrian Philipp <notifications@github.com> wrote:
> 
> In server.js:
> 
> > @@ -26,6 +16,12 @@ app.use(require("webpack-dev-middleware")(compiler, {
> >  }));
> >  app.use(require("webpack-hot-middleware")(compiler));
> >  
> > +//  Create hot-reloading bundle for server rendering
> > +var basePath = require('path').resolve(__dirname, '.');
> > +var webpackIsomorphicTools = new WebpackIsomorphicTools(require('./webpack-isomorphic-tools'))
> > +    .development(true)
> > +    .server(basePath, function() { });
> No problem :)
> 
> —
> Reply to this email directly or view it on GitHub.
>